src/apps/material/material.test.ts

```diff
@@ -0,0 +1,78 @@
+describe("Material app e2e", () => {
+  it("Should render the material page as expected", () => {
+    cy.log("Does the Material have title?");
+    cy.get(".text-header-h1").should("have.length.gte", 1);
+
+    cy.log("Check that cover has a src");
+    cy.get(".cover")
+      .children("img")
+      .should(
+        "have.attr",
+        "src",
+        "https:res.cloudinary.com/dandigbib/image/upload/t_ddb_cover_large/v1605727140/bogportalen.dk/9781848485532.jpg"
+      );
+
+    cy.log("Does the material have favourite buttons?");
+    cy.get(".button-favourite").should(
+      "have.attr",
+      "aria-label",
+      "Add to favorites"
+    );
+
+    cy.log("Does the material have horizontal lines?");
+    cy.get(".horizontal-term-line").should(
+      "contain.text",
+      // TODO: The series string being rendered makes no sense with the dummy data given
+      // and the (lack) of knowledge of which properties to use and how to combine them.
+      // Therefor when data is in place we can revisit this part of the test.
+      "Nr 1  i serienDummy Some SeriesNr 1  i serienDummy Some SeriesI samme serie Dummy Some TitleDummy Some TitleDummy Some TitleDummy Some TitleDummy Some TitleDummy Some TitleDummy Some TitleEmneord Dummy Some fictional subjectDummy 1950-1980Dummy Jens JensenDummy Some CorporationEmnetal Dummy skønlitteratur"
+    );
+
+    cy.log("Does the material have authors?");
+    cy.get(".material-header__content p.text-body-large").should(
+      "contain.text",
+      "Af  Dummy Jens Jensen"
+    );
+
+    cy.log("Does a material have the expected number of availibility labels?");
+    cy.get(".material-header__availability-label")
+      .find("a")
+      .should("have.length", 7);
+  });
+
+  beforeEach(() => {
+    cy.visit("/iframe.html?args=&id=apps-material--material");
+
+    // Intercept graphql search query.
+    cy.fixture("material/fbi-api.json")
+      .then((result) => {
+        cy.intercept("POST", "**/opac/graphql**", result);
+      })
+      .as("Graphql search query");
+
+    // Intercept covers.
+    cy.fixture("material/cover.json")
+      .then((result) => {
+        cy.intercept("GET", "**/api/v2/covers?**", result);
+      })
+      .as("Cover service");
+
+    // Intercept availability's service.
+    cy.fixture("material/availability.json")
+      .then((result) => {
+        cy.intercept("GET", "**/availability/v3?recordid=**", result);
+      })
+      .as("Availability service");
+
+    // Intercept like button
+    cy.intercept(
+      "HEAD",
+      "https://prod.materiallist.dandigbib.org/list/default/work-of:870970-basis:52557240",
+      {
+        statusCode: 401
```

Comment:
Please reconsider this mock.

Is there a reason why we want to return 401 (Unauthorized) for this request? I would have expected a 404 (the material is not on the list) or a 200 (Material is on list).

Also there is no reason to use a very specific url. It will easily break if something else changes.